lib/cookies.js

```diff
@@ -50,9 +51,9 @@ Cookies.prototype = {
     index = this.keys.index(data, remote)
 
     if (index < 0) {
-      this.set(sigName, null, {path: "/", signed: false })
+      if (autoSet) this.set(sigName, null, {path: "/", signed: false })
     } else {
-      index && this.set(sigName, this.keys.sign(data), { signed: false })
+      if (autoSet) index && this.set(sigName, this.keys.sign(data), { signed: false })
```

Comment:
the idea behind this set is that if the latest `key` isn't being used, update the cookie to use the latest `key`. this allows you to rotate your `key`s overtime. thus, you should never have this `if` statement